src/js/iotjs.js

```diff
@@ -69,9 +69,6 @@ setInterval = _timeoutHandler.bind(this, 'setInterval');
 clearTimeout = _timeoutHandler.bind(this, 'clearTimeout');
 clearInterval = _timeoutHandler.bind(this, 'clearInterval');
 
-// Make sure that it is not accessible elsewhere
-delete _timeoutHandler;
```

Comment:
When you delete this line, we can access this variable anywhere. (related issue #816)
I suggest the following.
```js
(function() {
  var timers = undefined;

  var _timeoutHandler = function(mode) {
    if (timers == undefined) {
      timers = Native.require('timers');
    }
    return timers[mode].apply(this, Array.prototype.slice.call(arguments, 1));
  }

  global.setTimeout = _timeoutHandler.bind(this, 'setTimeout');
  global.setInterval = _timeoutHandler.bind(this, 'setInterval');
  global.clearTimeout = _timeoutHandler.bind(this, 'clearTimeout');
  global.clearInterval = _timeoutHandler.bind(this, 'clearInterval');
})();
```